Dokumentationen/test/test_cases.adoc

```diff
@@ -27,17 +27,78 @@ a| * Datenbankeintrag für die Bestellung ist angelegt
 * Dozent befindet sich auf der Buchungsbestätigungseite
 
 |*Benötigte Daten*
-a| * Datum und Uhrzeit, Studiengang
+a| * Valides Datum, valide Uhrzeit und valider Studiengang
 * es muss ein Dozentenkonto geben 
 
 |===
 
 ==== TC02 - Dozent kann keine invaliden Voreinstellungen auswählen 
 
+|===
+
+|*Beschreibung* |Der Dozent versucht bewusst falsche Eingaben in den Voreinstellungen zu treffen. Damit sind keine Menschlichenfehler gemeint, wie z.B. ein falsches Datum ausgewählt zu haben sondern ein Datum in der Vergangenheit zu wählen. 
+
+
+|*Vorbedingungen*
+a| * Dozent ist angemeldet  
+* Es wurde auf den Button "Buchung starten" geklickt
```

Comment:
Vielleicht noch ne generelle Frage an alle Beteiligten: Bei den Use Cases sollten wir ja möglich wenig Bezug auf das UI nehmen. Wie sieht es hier in den Test Cases aus? Sollten wir so etwas wie "Es wurde auf den Button xy geklickt" ersetzen durch z.B. "Es wurde eine Buchung begonnen"?

Reply:
Klingt logisch.

Reply:
Können wir machen